tests/github/export-diff.js

```diff
@@ -144,12 +172,14 @@ function getDiffMessage(plugin, fixtures) {
 
   for (const file of Object.keys(output.changedFiles)) {
```

Comment:
Please wrap this `for` loop in another `<blockquote>` tag to improve readability. See the PR description; I added the `<blockquote>` tag only for the first `<details>` tag.